### PR TITLE
Add support for IsReadOnly in DatePickerControl and DateTimePickerControl

### PR DESCRIPTION
**Pull request by michu**

### CODE REVIEW

File: src/Orc.Controls.Examples/Views/DatePicker.xaml [hunk 1]
```diff
@@ -12,7 +12,8 @@
     <StackPanel>
         <StackPanel Orientation="Horizontal">
             <TextBlock Text="DatePicker Control" VerticalAlignment="Center" Width="150"/>
-            <orc:DatePickerControl Value="{Binding DateValue}" Width="120" Margin="5" Format="yyyy/MM/dd" HorizontalAlignment="Left" />
+            <orc:DatePickerControl Value="{Binding DateValue}" Width="120" Margin="5" Format="yyyy/MM/dd" HorizontalAlignment="Left" IsReadOnly="{Binding IsChecked, ElementName=_readOnlyCheckbox}" />
+            <CheckBox x:Name="_readOnlyCheckbox" Content="Read only" IsChecked="False" Margin="15 9 8 8"/>
```
Comment (GeertvanHorrik) [inline src/Orc.Controls.Examples/Views/DatePicker.xaml]: Don't use underscores in element names please.

**Comment by GeertvanHorrik** [comment]
Please remove the underscores, then I can accept this one. Great work.

**Comment by michu** [comment]
So, how should I name this element ? ReadOnlyCheckBox or readOnlyCheckBox, which one is preferred by you ?

**Comment by GeertvanHorrik** [comment]
readOnlyCheckBox

**Comment by michu** [comment]
Renamed.

File: src/Orc.Controls.Examples/Views/TimeSpan.xaml [hunk 2]
```diff
@@ -13,7 +13,8 @@
         <StackPanel Orientation="Horizontal" VerticalAlignment="Top" >
             <TextBlock Text="TimeSpan Control" VerticalAlignment="Center" Width="150" />
             <orc:TimeSpanControl Width="120" Value="{Binding TimeSpanValue}"
-                                 Margin="5" HorizontalAlignment="Left"/>
+                                 Margin="5" HorizontalAlignment="Left" IsReadOnly="{Binding IsChecked, ElementName=_readOnlyCheckbox}" />
+            <CheckBox x:Name="_readOnlyCheckbox" Content="Read only" IsChecked="False" Margin="15 9 8 8"/>
```
Comment (GeertvanHorrik) [inline src/Orc.Controls.Examples/Views/TimeSpan.xaml]: this one as well

**Comment by michu** [comment]
Renamed, I'm sorry just forget about TimeSpanControl.

**Pull request merged**